app/src/main/res/drawable/selector_bookmark.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android" android:exitFadeDuration="@android:integer/config_mediumAnimTime">
-  <item android:state_pressed="false" android:state_focused="true" android:drawable="@drawable/ic_bookmark_selected" />
-  <item android:state_pressed="true" android:drawable="@drawable/ic_bookmark_selected" />
-  <item android:state_selected="true" android:drawable="@drawable/ic_bookmark_selected" />
-  <item android:drawable="@drawable/ic_bookmark_outline" />
+  <item android:state_pressed="false" android:state_focused="true" android:drawable="@drawable/ic_bookmark_grey600_24dp" />
```

Comment:
Ideally, I'd like to keep the naming convention. Could you rename your files to the same as the original ones?